exp/rendering-architecture.md

```diff
@@ -15,6 +17,16 @@ For AMD and Intel GPUs, Anbox Cloud uses the [Mesa driver](https://www.mesa3d.or
 
 See [Component versions](https://anbox-cloud.io/docs/component-versions) to refer to the actual version supported for any particular Anbox Cloud release.
 
+# Supported platforms
+
+For rendering, you can use the `swrast` or the `null` platforms depending on your requirements.
+
+`swrast` is a software rasterization platform, which is a rendering implementation of the Mesa driver with support for LLVMpipe. It can be utilised for use cases that require a visual output without a GPU. When you set `swrast` with LLVMpipe as your backend in the `session.yaml` file, the rendering pipeline uses this platform irrespective of any available GPUs. To know more about this implementation, see [LLVMpipe](https://docs.mesa3d.org/drivers/llvmpipe.html).
```

Comment:
`When you set `swrast` with LLVMpipe as your backend in the `session.yaml` file,`
This involves the technical details which we don't want our end user to dive into. maybe the following fits better.
```
When you launch an Anbox container with the `swrast` platform via:
    amc launch -p swrast <app_id>
```

`the rendering pipeline uses this platform irrespective of any available GPUs.`
->
The rendering pipe for the swrast or null platform has nothing different than the one for the webrtc platform with NVIDIA GPU support except for the fact that it's irrespective of any available GPUs